### PR TITLE
Fixed broken Firebase and DOM event handlers.

### PR DESCRIPTION
**Pull request by gmittal**
Trying to bring volunteer back into a semi-stable state.

@ankitr PTAL.

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -93,16 +99,19 @@ firebase.auth().signInWithPopup(provider).then(() => {
...
-                  $('.messageSpace')
-                    .append(`<div class="message from" id="${message}">${message.body}</div>`);
-                } else {
-                  $('.messageSpace')
-                    .append(`<div class="message to" id="${message}">`
-                          + `${s.val()[message].body}</div>`);
-                }
-              });
+              if (s.val() !== null) {
+                console.log(s.val());
```
Comment (ankitr) [inline static/js/main.js]: Remove this.

**Review by ankitr — DISMISSED** [review]

File: static/js/main.js (hunk 1 above)
Comment (gmittal) [reply static/js/main.js]: I'm removing the console.log, but the s.val() !== null was added because it breaks otherwise.

Comment (gmittal) [reply static/js/main.js]: If the Firebase message object is empty, then the code breaks because it doesn't know how to iterate over an empty object.

**Review by ankitr — APPROVED** [review]
LGTM

**Pull request merged**